### DISCUSSION TITLE
Swapper (Neutral)

### DISCUSSION THREAD

**Discussion by Alf21** [Alf21/TTT2#790] [Role Ideas]
**Description**: The Swapper is a role that plays like the Jester (can't deal damage, and wants others to kill them), except once someone kills the Swapper, both the Swapper and their murderer revive, but their roles are swapped.

**Win Condition**: The Swapper cannot win. The goal of The Swapper is to pass their role onto someone else.

Long story short, The Swapper is the same thing as The Swapper from the TTT1 [Custom Roles](https://steamcommunity.com/sharedfiles/filedetails/?id=1215502383) mod along with [this](https://steamcommunity.com/sharedfiles/filedetails/?id=2112620831). It's the only role that Custom Roles has that TTT2 doesn't, so someone making a Swapper for TTT2 would be great. 

Idea by: @SmungDaal

**Comment by TaintedEnergy** [comment]
This role has been created: https://steamcommunity.com/sharedfiles/filedetails/?id=2605759027&searchtext=ttt
